Roles/Crewmate/President.cs

```diff
@@ -222,7 +222,7 @@ public static void ReceiveRPC(MessageReader reader, PlayerControl pc, bool isEnd
             else CheckPresidentReveal.Add(PlayerId, false);
             return;
         }
-        EndMsg(pc, $"/finish");
+        EndMsg(pc, $"/finish|/总统结束|/总统结束会议");
```

Comment:
No, here should be only `EndMsg(pc, $"/finish");`
This code is needed exclusively for Modded clients so that they can reproduce the ability